diff --git a/libmspack/Makefile.am b/libmspack/Makefile.am
index b50404c..627ec62 100644
--- a/libmspack/Makefile.am
+++ b/libmspack/Makefile.am
@@ -89,11 +89,11 @@ test_chmd_order_SOURCES =	test/chmd_order.c test/md5.c test/md5.h \
 test_chmd_order_LDADD =		libmschmd.la
 test_chmd_test_SOURCES =	test/chmd_test.c libmschmd.la
 test_chmd_test_LDADD =		libmschmd.la
-test_chmd_test_DEPENDENCIES = 	libmschmd.la test/test_files/chmd/cve-2015-4467-reset-interval-zero.chm
+test_chmd_test_DEPENDENCIES = 	libmschmd.la $(srcdir)/test/test_files/chmd/cve-2015-4467-reset-interval-zero.chm
 test_chminfo_SOURCES =		test/chminfo.c libmschmd.la
 test_chminfo_LDADD =		libmschmd.la
 test_kwajd_test_SOURCES =	test/kwajd_test.c libmspack.la
 test_kwajd_test_LDADD = 	libmspack.la
 
-test/test_files/chmd/cve-2015-4467-reset-interval-zero.chm: test/test_files/chmd/cve-2015-4467-reset-interval-zero.chm.LZXC-is-lzxc
+$(srcdir)/test/test_files/chmd/cve-2015-4467-reset-interval-zero.chm: $(srcdir)/test/test_files/chmd/cve-2015-4467-reset-interval-zero.chm.LZXC-is-lzxc
 	sed s/lzxc/LZXC/ $< >$@
